caput/scripts/runner.py

```diff
@@ -1,16 +1,11 @@
-from os.path import (
-    normpath,
-    expanduser,
-    expandvars,
-    realpath,
-    abspath,
-    isabs,
-    join,
-    exists,
-)
+import os
+import sys
+
+from pathlib import Path, PurePath
```

Comment:
You probably don't need to be importing `PurePath` directly here. Removing the import will help identify places you should be calling methods on the objects directly.